main.go

```diff
@@ -56,6 +58,10 @@ const (
 	PGDUMP string = "pgdump"
 	// POSTGRES is the driver name for PostgreSQL.
 	POSTGRES string = "postgres"
+	// MYSQLDUMP is the driver name for mysql dump.
```

Comment:
These are mostly used only in `main.go`. If we put it in respective packages, then at every place in `main.go` we would need to use it as `eg: postgres.POSTGRES, postgres.PGDUMP, mysql.MYSQL, mysql.MYSQLDUMP`. 
Except in `report.go` file, I don't think they would be used anywhere inside packages. It would not be a good idea to move it to packages.

Reply:
If you still think that it should be moved to packages, then please let me know. I will make the changes

Reply:
I hope to avoid the cases for string comparison in `report.go`, such as `x == "mysql"` and `x == "postgres"`, because it is very easy to get misspelling and hard to detect the errors.

Reply:
@nevinheintze What is your opinion on this?

Reply:
These driver names are only used in `internal/report.go` except `main.go`. For sure we can create `driver.go` file in put it in `internal` package, but then the only benefit would be that we can use that constants at couple of places in report.go file. And also in main.go we would have to use it as internal.POSTGRES, internal.MYSQLDUMP etc.

Also, for 'isDump := (driverName == PGDUMP) || (driverName == MYSQLDUMP)' point - @hengfengli had suggested to check for 'dump' in drivername directly to avoid using driver names as they are not constants. Hence, I changed it accordingly.

Please let me know what do you recommend? If you think it's worth changing it then I will go ahead and make these changes :

- Create `driver.go` file in `internal` package and define constants. 

- Change variables in main.go file from `POSTGRES` --> `internal.POSTGRES` etc.

- Change `if strings.Contains(driverName, "dump")` to `isDump := (driverName == PGDUMP) || (driverName == MYSQLDUMP) ` in `report.go`.

Reply:
Yes, I kind of agree none of the options are really a big improvement.
Let's leave as is.